generated/nidcpower/ctypes_types.py

```diff
@@ -0,0 +1,69 @@
+import ctypes
+
+
+class ViStatus_ctype(ctypes.c_long):  # noqa: N801
+    pass
+
+
+class ViRsrc_ctype(ctypes.c_char_p):  # noqa: N801
+    pass
+
+
+class ViSession_ctype(ctypes.c_ulong):  # noqa: N801
+    pass
+
+
+class ViChar_ctype(ctypes.c_char):  # noqa: N801
+    pass
+
+
+class ViUInt32_ctype(ctypes.c_ulong):  # noqa: N801
+    pass
+
+
+class ViInt32_ctype(ctypes.c_long):  # noqa: N801
+    pass
+
+
+class ViInt16_ctype(ctypes.c_short):  # noqa: N801
+    pass
+
+
+class ViUInt16_ctype(ctypes.c_ushort):  # noqa: N801
+    pass
+
+
+class ViInt64_ctype(ctypes.c_longlong):  # noqa: N801
```

Comment:
Should we add ViUInt64 as well?

Reply:
Not needed yet... would rather not add untestable code.